### DISCUSSION TITLE
Update-VSTeamReleaseDefinition using -InFile or -ReleaseDefinition changes LinkBoardsWorkItems type. Maybe I'm doing something wrong?

### DISCUSSION THREAD

**Discussion by klabranche** [DarqueWarrior/vsteam#540] [Q&A]
I am writing a PS script that is finding release definitions (classic) that are using a variable group and then updating the definition to use another variable group.

Basic flow for update is:

Get-VSTeamReleaseDefinition -Id # -JSON OR -Raw
update the variable group in question 
Update-VSTeamReleaseDefinition -InFile OR -ReleaseDefinition where I use ConvertTo-Json -Depth 100 for both.

I get the expected changes in the release definition.  The revision #, modified on, the variable group id changed in the script.

I also unexpectedly get a change:
        "LinkBoardsWorkItems": {
          "$type": "System.Boolean",
          "$value": false
        }
Is changed to:
        "LinkBoardsWorkItems": {
          "$type": "System.String",
          "$value": "False"
        }

Wondering if I'm doing something wrong in the conversion to/from JSON or if this is expected to occur?
I'm using 7.9 atm but looked at the updates and none seemed to affect this area so haven't updated.
The release still works so it doesn't appear to be a breaking issue.

**Comment by klabranche** [comment]
Also noticed that if I update from the GUI, it puts the LinkBoardsWorkItems type back to boolean and false....

**Comment by SebastianSchuetze** [comment]
I think you are not doing anything wrong. Class build and releases with their JSON are not well documented and are not further developed. The cmdlets are practically only pushing the JSON to the API. 
It might have to do something with the JSON content itself that the module is not handling. It will also not handle since classic builds and releases are not recommended anymore.
